server/data/services/githubService.js

```diff
@@ -0,0 +1,25 @@
+const axios = require('axios');
+
+class GithubService {
```

Comment:
Same points I wrote in `AuthController`, this is a "class" here, but doesn't need to be